README.md

```diff
@@ -47,6 +47,16 @@ $ cargo build --release
 
 This will create the `lucky_commit` binary (`lucky_commit.exe` on Windows) in the `target/release` directory. You can move this to wherever you want, or set up an alias for it.
 
+#### Distro packages
+
+##### Arch Linux
+
+`lucky-commit` can be installed from the [community repository](https://archlinux.org/packages/community/x86_64/lucky-commit/) using [pacman](https://wiki.archlinux.org/title/Pacman):
+
+```
+$ pacman -S lucky-commit
```

Comment:
If I'm reading [this](https://github.com/archlinux/svntogit-community/blob/216c305a8e5bfcd050eeba0aad91852032e9e074/repos/community-x86_64/PKGBUILD#L23) correctly, would Arch Linux users who install with this command get the CPU-only version? That's fine, but perhaps it would be good to note this.

Incidentally, my understanding is that if the `ocl-icd-opencl-dev` package is installed on Linux, then linking will succeed, regardless of whether the machine has GPUs. Once it builds successfully, [`lucky-commit` will detect the presence or absence of GPUs at runtime](https://github.com/not-an-aardvark/lucky-commit/blob/000000000e8de4d8848f10f4f14b942d2cc76072/src/lib.rs#L187-L192), and use this to decide whether to use the GPU or CPU version. This would be useful because it would allow users with GPUs to get the fast version automatically, while still working fine for users without GPUs.

AFAIK it's not trivial to specify system library dependencies in a cross-platform way from my cargo crate, but maybe it's easier to do that from an Arch Linux package. (That said, you know much more about linux packaging than I do, so the decision is up to you.)

Reply:
Thanks for letting me now about this. I enabled the OpenCL support in [2.0.5-2](https://github.com/archlinux/svntogit-community/commit/c909babd71fc5a79fb73d903359d9aba3ca4b4c2#diff-3e341d2d9c67be01819b25b25d5e53ea3cdf3a38d28846cda85a195eb9b7203a). This will require [ocl-icd](https://archlinux.org/packages/extra/x86_64/ocl-icd/) package to be installed and `lucky-commit` will decide whether to use the GPU or CPU version at runtime like you said. Really cool feature 🥂